README.md

```diff
@@ -34,6 +34,7 @@ Elyra currently includes:
 * [Notebook versioning based on git integration](#notebook-versioning-based-on-git-integration)
 * [Notebook navigation using auto-generated **Table of Contents**](#notebook-navigation-using-auto-generated-table-of-contents)
 * [Reusable configuration for runtimes](#reusable-configuration-for-runtimes)
+* [Add custom Code Snippets - Experimental](#code-snippets)
```

Comment:
I would move this to right after `Python script...` and would call it `Reusable code snippets (experimental)`